account_invoice_tax_note/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Account invoice tax note",
+    'version': "11.0.1.0.0",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "summary": "In some situations, a mention must be displayed on invoices "
+               "when a specific tax is used.\nThis module lets you define such"
+               " a mention on Tax Groups.\nThis mechanism complements the "
+               "invoice note coming from the fiscal position.",
```

Comment:
This is a bit long for the summary. I'd say "Print tax notes on customer invoices"